face_detection_masking/constants.py

```diff
@@ -2,7 +2,7 @@
     Google Vision
 """
 EMOTION_THRESHOLD = 3
-LIGHTING_THRESHOLD = 3
+LIGHTING_THRESHOLD = 2
 BLURRY_THRESHOLD = 2
 ANGLE_THRESHOLD = 10.0
```

Comment:
Can you potentially add some comments to explain what these constants are used. It's not mandatory, just good practice.

Reply:
I am not exactly sure how these constants were chosen, but they are used to validate the face image